### PR TITLE
Move gcs to gcs module

### PR DESCRIPTION
**Pull request by patflynn**
More progress towards #1896 

Pulls GCS feature-set out to its own module.  Figured I'd pick something easy first to verify that the approach works.

### CODE REVIEW

File: google-cloud-tools-plugin/google-cloud-storage/src/com/google/cloud/tools/intellij/gcs/GoogleCloudStorageMessageBundle.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+package com.google.cloud.tools.intellij.gcs;
+
+import com.intellij.CommonBundle;
+import java.util.ResourceBundle;
+import org.jetbrains.annotations.NonNls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.PropertyKey;
+
+public class GoogleCloudStorageMessageBundle {
```
Comment (etanshaul) [inline google-cloud-tools-plugin/google-cloud-storage/src/com/google/cloud/tools/intellij/gcs/GoogleCloudStorageMessageBundle.java]: nit: but for consistency with `GctMessageBundle`, maybe `GcsMessageBundle`? I'm fine renaming the other one too.

File: google-cloud-tools-plugin/google-cloud-core/resources/messages/CoreMessageBundle.properties [hunk 2]
```diff
@@ -33,4 +33,6 @@ error.dialog.disable.plugin.restart=Changes in plugin configuration will take ef
 error.dialog.disable.plugin.norestart=After disabling the plugin please restart {0} to apply changes in plugin configuration.
 error.dialog.disable.plugin.title=Disable Plugin
 error.dialog.disable.plugin.action.disable=&Disable
-error.dialog.disable.plugin.action.disableAndRestart=Disable and &Restart
\ No newline at end of file
+error.dialog.disable.plugin.action.disableAndRestart=Disable and &Restart
+
+cloud.tools.group.menu.text=Google Cloud Tools
```
Comment (etanshaul) [inline google-cloud-tools-plugin/google-cloud-core/resources/messages/CoreMessageBundle.properties]: I wonder if this needs to be named differently than the one in the other module. i.e. do the group names here need to be unique

Comment (patflynn) [reply google-cloud-tools-plugin/google-cloud-core/resources/messages/CoreMessageBundle.properties]: hmm I think this was mistakenly added  here.

File: google-cloud-tools-plugin/google-cloud-storage/src/com/google/cloud/tools/intellij/gcs/GoogleCloudStorageMessageBundle.java (hunk 1 above)
Comment (patflynn) [reply google-cloud-tools-plugin/google-cloud-storage/src/com/google/cloud/tools/intellij/gcs/GoogleCloudStorageMessageBundle.java]: I'm being consistent with core. GCT should cease to exist when all the modules are extracted out.

Comment (etanshaul) [reply google-cloud-tools-plugin/google-cloud-storage/src/com/google/cloud/tools/intellij/gcs/GoogleCloudStorageMessageBundle.java]: sgtm

File: google-cloud-tools-plugin/google-cloud-core/resources/messages/CoreMessageBundle.properties (hunk 2 above)
Comment (patflynn) [reply google-cloud-tools-plugin/google-cloud-core/resources/messages/CoreMessageBundle.properties]: ah yeah, cause I was experimenting with pulling the action registration on the menu into the GCS plugin.xml but that was less straight forward than I expected.  I'm going to put this message back in GCT.

File: google-cloud-tools-plugin/google-cloud-storage/build.gradle [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        exclude group: 'com.google.guava', module: 'guava'
+        exclude group: 'com.google.oauth-client', module: 'google-oauth-client'
+        exclude group: 'com.google.protobuf', module: 'protobuf-java'
+        exclude group: 'com.google.protobuf', module: 'protobuf-java-util'
+    }
+
+    compileOnly 'com.google.auto.value:auto-value:1.4.1'
+    apt         'com.google.auto.value:auto-value:1.4.1'
+
+    testCompile 'xalan:serializer:2.7.1'
```
Comment (etanshaul) [inline google-cloud-tools-plugin/google-cloud-storage/build.gradle]: did you find out that these are needed by just running it and observing the errors? Just making sure it wasn't added unnecessarily.

File: google-cloud-tools-plugin/google-cloud-core/resources/messages/CoreMessageBundle.properties (hunk 2 above)
Comment (patflynn) [reply google-cloud-tools-plugin/google-cloud-core/resources/messages/CoreMessageBundle.properties]: fixed.

File: google-cloud-tools-plugin/google-cloud-storage/build.gradle (hunk 3 above)
Comment (patflynn) [reply google-cloud-tools-plugin/google-cloud-storage/build.gradle]: good point. I didn't need it. removed

**Pull request merged**